templates/content/field-group-html-element--node--oe-project--group-result.html.twig

```diff
@@ -11,6 +11,9 @@
     {% if title %}
       <h2 class="ecl-u-type-heading-2">{{ title }}</h2>
     {% endif %}
-    {{ children }}
+    {{ element.oe_project_results }}
   </div>
 </div>
+<div id="project-result-files">
+  {{ element.oe_project_result_files }}
+</div>
```

Comment:
Try this:

```twig
<div id="project-results">
  <div class="ecl-row">
    <div class="ecl-col-6">
      {% if title %}
        <h2 class="ecl-u-type-heading-2">{{ title }}</h2>
      {% endif %}
      {{ element.oe_project_results }}
    </div>
  </div>
  {{ element.oe_project_result_files }}
</div>
```

It will make it easier to port tests both here and on EWCMS.